### PR TITLE
Run make fmt

### PR DESCRIPTION
**Pull request by anuraaga**
When doing some hacking, ran `make fmt` and found these unrelated changes. Isn't formatting checked as part of CI?

### CODE REVIEW

File: receiver/elasticsearchreceiver/integration_test.go [hunk 1]
```diff
@@ -24,12 +24,13 @@ import (
...
 	"time"
 
-	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest/golden"
 	"github.com/stretchr/testify/require"
 	"github.com/testcontainers/testcontainers-go"
 	"github.com/testcontainers/testcontainers-go/wait"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest/golden"
```
Comment (dmitryax) [inline receiver/elasticsearchreceiver/integration_test.go:32]: This part should be moved below I believe

File: receiver/rabbitmqreceiver/integration_test.go [hunk 2]
```diff
@@ -20,16 +20,18 @@ package rabbitmqreceiver
 import (
 	"context"
 	"fmt"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest"
 	"path/filepath"
 	"testing"
 	"time"
 
-	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest/golden"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest"
```
Comment (dmitryax) [inline receiver/rabbitmqreceiver/integration_test.go:27]: This should be part of the third section

File: receiver/elasticsearchreceiver/integration_test.go (hunk 1 above)
Comment (anuraaga) [reply receiver/elasticsearchreceiver/integration_test.go:33]: Oops - thanks for checking think I fixed them

**Comment by mx-psi** [comment]
> When doing some hacking, ran `make fmt` and found these unrelated changes. Isn't formatting checked as part of CI?

These are all either Windows files or test files. It's possible that we are only running this on Linux

**Review by djaglowski — APPROVED** [review]

File: receiver/elasticsearchreceiver/integration_test.go [hunk 3]
```diff
@@ -24,14 +24,13 @@ import (
 	"testing"
 	"time"
 
-	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest/golden"
 	"github.com/stretchr/testify/require"
-	"github.com/testcontainers/testcontainers-go"
```
Comment (djaglowski) [inline receiver/elasticsearchreceiver/integration_test.go:30]: This should not be removed.

File: receiver/mongodbreceiver/integration_test.go [hunk 4]
```diff
@@ -24,15 +24,14 @@ import (
 	"testing"
 	"time"
 
-	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest/golden"
 	"github.com/stretchr/testify/require"
-	"github.com/testcontainers/testcontainers-go"
```
Comment (djaglowski) [inline receiver/mongodbreceiver/integration_test.go:30]: Same

File: receiver/rabbitmqreceiver/integration_test.go [hunk 5]
```diff
@@ -20,18 +20,17 @@ package rabbitmqreceiver
...
 	"context"
 	"fmt"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest"
 	"path/filepath"
 	"testing"
 	"time"
 
-	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/scrapertest/golden"
 	"github.com/stretchr/testify/require"
-	"github.com/testcontainers/testcontainers-go"
```
Comment (djaglowski) [inline receiver/rabbitmqreceiver/integration_test.go:30]: Same

**Review by djaglowski — COMMENTED** [review]
I thought this was good and only needed a `gotidy`, so I ran that and pushed. 

However, `testcontainers-go` is required in the three integration tests, so this needs to be readded.

Since I already got myself involved in this, I'll push this change.

**Review by codeboten — APPROVED** [review]

**Pull request merged**